### DISCUSSION TITLE
多翻译源同时翻译

### DISCUSSION THREAD

**Discussion by Geo123abc** [immersive-translate/immersive-translate#800] [Ideas]
### 新功能 | New Feature

1. 可以选择多个翻译源同时进行翻译

### 为什么您想要添加这些功能 | Why you want to add these features

1. 方便对比各个翻译服务的准确性；
2. 针对不同行业，可能不同翻译服务的准确性有不同，如果用户起初使用多个翻译源，在使用过程中可以对效果进行对比，然后再选择取舍。

### 提议 | Proposal

在这里看到的

https://www.v2ex.com/t/961501#reply13

**Comment by orangekana** [comment]
同样建议这一功能。
1. 对比各个翻译服务的准确性，可避免被单个翻译服务的错误误导；
2. 方便作为翻译辅助，为找到更好的翻译方式提供灵感

参考：https://www.machinetranslation.com/ 同时展示多个模型的翻译结果

<img width="813" height="464" alt="135c4fb0a919300b1fc995d6920ad19c" src="https://github.com/user-attachments/assets/e10208c0-12a1-4f6a-b636-42e5e48015fd" />

<img width="795" height="529" alt="fbb1f13e4ddae526965e1b11f46e16da" src="https://github.com/user-attachments/assets/1f369730-0d05-489f-a9b0-20b6917342ec" />
